src/GafferImage/OpenImageIOReader.cpp

```diff
@@ -916,7 +916,7 @@ void OpenImageIOReader::affects( const Gaffer::Plug *input, AffectedPlugsContain
 		outputs.push_back( tileBatchPlug() );
 	}
 
-	if( input == fileNamePlug() || input == refreshCountPlug() || input == missingFrameModePlug() )
+	if( input == fileNamePlug() || input == refreshCountPlug() || input == missingFrameModePlug() || input == tileBatchPlug() )
```

Comment:
No, not necessary for later commits.  I was just looking through this code for my stuff, and noticed the inaccuracy.  Maybe it's better to ignore this for now since it's working?

Reply:
Yeah, I think maybe it's best to ignore for now - I suspect there's a bigger cleanup that could be made, but that's out of scope for this PR.